resolwe/flow/executors/__init__.py

```diff
@@ -122,6 +123,14 @@ def run(self, data_id, script, verbosity=1):
 
         self.data_id = data_id
 
+        # Fetch data instance to get any executor requirements.
+        if data_id != 'no_data_id':
```

Comment:
I would wait for #187 to be merged as it introduces some refactoring around the Docker test utilities.